src/panels/lovelace/common/validate-condition.ts

```diff
@@ -15,13 +17,50 @@ export function checkConditionsMet(
     const state = hass.states[c.entity]
       ? hass!.states[c.entity].state
       : UNAVAILABLE;
+    if (!c.operator) {
+      if (c.state && state !== c.state) {
+        return false;
+      }
+      if (c.state_not && state === c.state_not) {
+        return false;
+      }
+      return true;
+    }
 
-    return c.state ? state === c.state : state !== c.state_not;
+    if (!c.value) {
+      // error state must be defined when operator is defined too
+      return false;
+    }
+
+    if (c.operator === "==") {
```

Comment:
Can we reuse evaluateFilter <https://github.com/home-assistant/frontend/blob/dev/src/panels/lovelace/common/evaluate-filter.ts#L3>?

Reply:
I can try it. But how are there handled numbers? I do not see conversion from string to number. Or it is not needed?

Reply:
It should already be a number? We should not need to convert it

Reply:
done, thanks :)